charts/plugin-br-pix-switch/templates/cob-hub/deployment.yaml

```
@@ -0,0 +1,92 @@
+{{- if .Values.cobHub.enabled }}
+{{- $component := "cob-hub" }}
+{{- $values := .Values.cobHub }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "plugin-br-pix-switch.componentFullname" (dict "context" $ "component" $component) }}
+  labels:
+    {{- include "plugin-br-pix-switch.labels" (dict "context" $ "component" $component) | nindent 4 }}
+spec:
+  revisionHistoryLimit: {{ $values.revisionHistoryLimit | default 10 }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)